Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:stretch-curl
+FROM docker.io/node:9-stretch
```

Comment:
I've never seen the docker.io big before - can you explain it?

Reply:
Yeah.  Basically, you can have multiple Docker registries out there.  `docker.io` is the main one, and technically it's the default, but this prevents any name conflicts and also signifies that we're using Docker's `node` version (rather than anything that you might have tagged as `node` locally.

So, say, we wanted to use `quay.io` for hosting or `docker.cs.stolaf.edu`... (which is a thing, kinda) that's where that would go.